.github/workflows/ci.yml

```diff
@@ -69,6 +69,7 @@ jobs:
       - uses: ./.github/actions/setup
         with:
           java-version: ${{ matrix.java-version }}
+          cleanup-node: ${{ matrix.java-version == '22-ea' && 'true' || 'false' }}
```

Comment:
What is && true for?

Also, why not cleanup always?

Reply:
I'll cleanup always for maven-checks